distributed/comm/core.py

```diff
@@ -238,7 +238,7 @@ def resolve_address(addr):
 
 
 @gen.coroutine
-def connect(addr, timeout=3, deserialize=True):
+def connect(addr, timeout=3, deserialize=True, ssl_options=None):
```

Comment:
Yes, but the `distributed.comm` layer abstracts away the transport mechanism, so you cannot assume that the notion of SSL options has a meaning for transports other than TCP.